packages/diffast-etc/diffast-etc.0.1.1/opam

```diff
@@ -0,0 +1,46 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Diff/AST etc files"
+description:
+  "Config files for Diff/AST. Diff/AST is a fine-grained source code differencing tool based on an algorithm for computing tree edit distance (TED) between two ordered labeled trees."
```

Comment:
Could you help me understand why a user would want to install the config fils at https://github.com/codinuum/diffast/tree/main/src/etc as a separate package?

Reply:
Several subpackages (we will PR later) such as parser CLI subpackages share the etc subpackage. A user (or a meta package) can select subpackages like SPL (software product line).